src/couch_replicator_scheduler.erl

```diff
@@ -226,16 +233,21 @@ oldest_job_first(#job{} = A, #job{} = B) ->
 
 not_recently_crashed(#job{} = Job) ->
     case crash_history(Job) of
-        [] ->
-            true;
-        [{crashed, When} | _] ->
+        [{{crashed, _Reason}, When} | _] = Crashes ->
+            BackoffExp = erlang:min(length(Crashes), ?MAX_BACKOFF_EXPONENT),
```

Comment:
I think random would make sense if the backoff exponent was persisted in memory. If the backoff was randomly generation for each filtration of crashed jobs, jobs will be biased toward low exponents.